web/src/main/java/org/mskcc/cbio/oncokb/config/MvcConfiguration.java

```diff
@@ -71,6 +102,85 @@ public ServletContextInitializer sentryServletContextInitializer() {
         return new io.sentry.spring.SentryServletContextInitializer();
     }
 
+    private ApiInfo getDefaultApiInfo(String title, String description, String apiVersion) {
+        return new ApiInfo(
+            Optional.ofNullable(title).orElse("OncoKB APIs"),
+            Optional.ofNullable(description).orElse(""),
+            Optional.ofNullable(apiVersion).orElse("v0.0.1"),
+            "https://www.oncokb.org/terms",
+            new Contact("OncoKB", "https://www.oncokb.org", "contact@oncokb.org"),
+            "Terms of Use",
+            "https://www.oncokb.org/terms"
+        );
+    }
+
+    private void updateDocketHost(Docket docket, ServletContext servletContext, String basePath) {
+        // Update swagger host/protocal with environment variable
+        String swaggerUrl = PropertiesUtils.getProperties(SWAGGER_URL);
+        if (StringUtils.isNotEmpty(swaggerUrl)) {
+            URL url = null;
+            try {
+                url = new URL(swaggerUrl);
+            } catch (MalformedURLException e) {
+                e.printStackTrace();
+            }
+            if (url != null) {
+                docket
+                    .pathProvider(new RelativePathProvider(servletContext) {
+                        @Override
+                        public String getApplicationBasePath() {
+                            return basePath;
+                        }
+                    })
+                    .protocols(Collections.singleton(Optional.ofNullable(url.getProtocol()).orElse("http")))
+                    .host(url.getHost());
```

Comment:
@calvinlu3 this is a bit different from what you implemented in https://github.com/oncokb/oncokb/pull/3287. Let me know how you think.